worker/undertaker/undertaker.go

```diff
@@ -172,7 +175,10 @@ func (u *Undertaker) processDyingModel() error {
 				// destroy any remaining environ resources.
 				return nil
 			}
-			// Yes, we ignore the error. See comment above.
+			// Yes, we ignore the error. See comment above. But let's at least
+			// surface it in status.
+			u.setStatus(status.Destroying, fmt.Sprintf("%d attempt to destroy model:  %v", attempt, err))
```

Comment:
2 attempt to destroy model: error
would probably read better as:
"attempt %d to destroy model failed (will retry): %v"

moving the counter helps readability, and some indication as to what we are doing about this failure.
These are going to be shown in "juju status", so making them legible is important.